front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -338,7 +337,7 @@ const ManchetteWithSpaceTimeChartWrapper = ({
       !draggingState &&
       selectedTrainScheduleId &&
       hoveredItem &&
-      'pathId' in hoveredItem.element
+      isSegmentPickingElement(hoveredItem.element)
```

Comment:
Ditto

Reply:
yes, I had just pushed that change, perhaps the rebase messed up github and you were looking at an older diff

Reply:
I see the change for my previous comment, but it seems this one still needs to be addressed?

Reply:
Everything looks good now, thanks!

Reply:
ok, now it’s pushed, sorry